resample/version.py

```diff
@@ -0,0 +1 @@
+version = "0.21"
```

Comment:
Should we just bump this to 1.0.0? The current version number is essentially meaningless.

Reply:
Yes, once we finished this interface overhaul, I am all for changing to 1.0.0!